src/ray/core_worker/transport/transport.h

```diff
@@ -32,7 +32,8 @@ class CoreWorkerTaskSubmitter {
 /// This class receives tasks for execution.
 class CoreWorkerTaskReceiver {
  public:
-  using TaskHandler = std::function<Status(const TaskSpecification &task_spec)>;
+  using TaskHandler = std::function<Status(
+      const TaskSpecification &task_spec, std::vector<std::shared_ptr<Buffer>> *results)>;
```

Comment:
Change it to `std::vector<std::shared_ptr<RayObject>>` so we can write metadata to plasma store.

Reply:
sure.